schemas/README.md

```diff
@@ -0,0 +1,76 @@
+## MIPS Submission Schemas
```

Comment:
So this README gets removed in a subsequent PR. It is duplicative. and we remove the schemas folder and replace it with the measures folder to follow CMS standards/naming.